response_operations_ui/templates/collection_exercise/ce-sample-ci-section.html

```diff
@@ -58,24 +67,32 @@
                             }
                         ]
                     }
-                ]
-            })
-        }}
+                ] %}
+            {% do surveyTableData | setAttribute("trs", surveyTableDataRows) %}
+        {% endif %}
+    
+        {{ onsTable(surveyTableData) }}
+    
+        {% if 'INIT' == sample.state %}
+              <div class="ons-u-mb-m">
+                  <div class="ons-u-fs-r">
+                      Loading&hellip; <span class="ons-u-ml-xs"><a href="/surveys/{{survey.shortName}}/{{ce.exerciseRef}}/view-sample-ci?show_msg=true">Refresh to see progress</a></span>
+                  </div>
+              </div>
+        {% elif 'FAILED' == sample.state %}
```

Comment:
This does produce an extra line between the error banner and the button, which makes it look a bit odd